hack/src/verify_yaml.rs

```diff
@@ -0,0 +1,117 @@
+use cincinnati::plugins::internal::openshift_secondary_metadata_parser::plugin;
+use regex::Regex;
+use semver::Version;
+use serde::de::DeserializeOwned;
+use serde_yaml;
+use std::collections::HashSet;
+use std::path::PathBuf;
+use tokio;
+
+use anyhow::Context;
+use anyhow::Result as Fallible;
+
+pub async fn run() -> Fallible<HashSet<Version>> {
+  let data_dir = PathBuf::from("..");
+  let extension_re = Regex::new("ya+ml")?;
+  // Collect a list of mentioned versions
+  let mut found_versions: HashSet<Version> = HashSet::new();
+
+  println!("Verifying blocked edge files are valid");
+  let blocked_edge_path = data_dir.join(plugin::BLOCKED_EDGES_DIR).canonicalize()?;
+  let blocked_edge_vec =
+    walk_files::<plugin::graph_data_model::BlockedEdge>(&blocked_edge_path, &extension_re).await?;
```

Comment:
Do we need local walkers for validation?  I was expecting us to be able to wash through [the `github-secondary-metadata-scrape` plugin][1].

[1]: https://github.com/openshift/cincinnati/tree/3f5ea5804d1fe67f657e900d652c75622480f9ab/cincinnati/src/plugins/internal/graph_builder/github_openshift_secondary_metadata_scraper

Reply:
Somewhat in this space, and feel free to punt as out of scope for a v1 PR, but it would be nice to be able to feed this tool graph-builder and policy-engine plugin configs, which we could copy/edit/paste from our production update service deployment, and have it ingest the local graph-data checkout, wash through the rest of the plugin stack, and spit out assorted policy-engine output graphs for all architectures and channels.  Or diff vs. the current production graph.  Or something...

Reply:
metadata-scrape is too forgiving, it intentionally skips files with no extension etc. Perhaps later we could introduce "strict" mode there and merge the walkers.

>it would be nice to be able to feed this tool graph-builder and policy-engine plugin configs

That would be useful to setup configuration (if the registry or org name changes). Probably out of scope now

> spit out assorted policy-engine output graphs for all architectures and channels

that could be done, but its not clear what's the expected graph is. Every change would produce a diff, but its not trivial to validate it. Its out of scope for v1, but might become useful

Reply:
@wking I would've gone in a similar direction to yours but I never got around to pursuing the implementation. My idea was to make a CLI tool very similar to what the GB and PE are today, but without the looping and serving aspects. It would just run the plugin chain once and yield the graph output. The latter could be handled via a plugin as. It might actually be pretty easy to add a config option such as `runonce` to the graphbuilder and add a small plugin which emits the graph to disk, stdout, or whatever we desire.

Reply:
@wking I came up with https://github.com/openshift/cincinnati/pull/307 which allows the removal of the walker and error types in this PR. @vrutkovs and I decided that this is enough for this PR and would like to postpone further optimization. Is that fine with you?

Reply:
I am fine punting anything.